### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/215
Update README for more detailed information, and sync with current code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,104 @@
 # PaddlePaddle Cloud
 
-## Using Command-line To Submit Cloud Training Jobs
+PaddlePaddle Cloud is a Distributed Deep-Learning Cloud Platform for both cloud
+providers and enterprises.
+
+PaddlePaddle Cloud use [kubernetes](https://kubernetes.io) as it's backend job
```
Comment (Yancey1989) [inline README.md]: kubernetes => Kubernetes

File: README.md [hunk 2]
```diff
@@ -1,82 +1,104 @@
...
   - `spec.template.spec.containers[0].volumes` change the `hostPath` which match your data folder.
   - `spec.template.spec.nodeSelector.`, edit the value `kubernetes.io/hostname` to host which data folder on.You can use `kubectl get nodes` to list all the Kubernetes nodes.
 - Configure `settings.py`
   - Add your domain name to `ALLOWED_HOSTS`.
-  - Configure `DATACENTERS` to your backend storage.
-- Configure `cloud_ingress.yaml`
-  - `spec.rules[0].host` specify your domain name
+  - Configure `DATACENTERS` to your backend storage, supports CephFS and HostPath currently.
+    You can use HostPath mode to make use of shared file-systems like "NFS".
+- Configure `cloud_ingress.yaml` is your kubernetes cluster is using "ingress"
```
Comment (Yancey1989) [inline README.md]: `"ingress"` => `[ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)`

File: README.md [hunk 3]
```diff
@@ -1,82 +1,104 @@
...
   mkdir -p /home/pcloud/data/mysql
   mkdir -p /home/pcloud/data/certs
   ```
-  - Copy Kubernetes CA files (ca.pem, ca-key.pem, ca.srl) to `pcloud_data/certs` folder
-  - Copy Kubernetes admin user key (admin.pem, admin-key.pem) to `pcloud_data/certs` folder
-  - Copy CephFS Key file(admin.secret) to `pcloud_data/certs` folder
-  - Copy `/paddlecloud/settings.py` file to `pcloud_data` folder
+  - Copy Kubernetes CA files (ca.pem, ca-key.pem, ca.srl) to `/home/pcloud/data/certs` folder
+  - Copy Kubernetes admin user key (admin.pem, admin-key.pem) to `/home/pcloud/data/certs` folder
+  - Copy CephFS Key file(admin.secret) to `/home/pcloud/data/certs` folder
```
Comment (Yancey1989) [inline README.md]: CephFS key is also an optional file.

File: README.md (hunk 1 above)
Comment (typhoonzero) [inline README.md]: Done

File: README.md (hunk 3 above)
Comment (typhoonzero) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (typhoonzero) [inline README.md]: Put link in the first time when ingress appears.

**Pull request merged**